pkg/virt-controller/watch/snapshot/snapshot.go

```diff
@@ -657,6 +679,7 @@ func (ctrl *VMSnapshotController) updateSnapshotStatus(vmSnapshot *snapshotv1.Vi
 	} else if vmSnapshotReady(vmSnapshotCpy) {
 		updateSnapshotCondition(vmSnapshotCpy, newProgressingCondition(corev1.ConditionFalse, "Operation complete"))
 		updateSnapshotCondition(vmSnapshotCpy, newReadyCondition(corev1.ConditionTrue, "Operation complete"))
+		updateSnapshotCondition(vmSnapshot, newFreezingCondition(corev1.ConditionFalse, vmSnapshotSourceThawed))
```

Comment:
same here, also if the vm is still frozen because of some bug that the unfreeze never happened then this is actually wrong

Reply:
The idea is that the content will not be ready (hence the snapshot) until observed unfrozen

Reply:
thats the ideal haha but it might not be so - i.e bug